routes/contents.js

```diff
@@ -1,7 +1,20 @@
 const Router = require('koa-router');
+const Content = require('../models/Content');
+const DBMethods = require('../common/baseDBMethods')
 
 const router = new Router();
 
-// Add sub-routes here using router.get(...), router.post(...), etc.
+router.get('/', async (ctx) => {
+        await DBMethods.list(ctx, Content);   
```

Comment:
Can we make the functions return a function specific to the model?
If so, we won't have to create an arrow function here and will just pass the specific function.